Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx

```diff
@@ -242,35 +242,35 @@
     <comment>Text for Hour</comment>
   </data>
   <data name="HoursIntervalTypeReminder" xml:space="preserve">
-    <value>Heures</value>
+    <value>Intervalle d'heures pour le rappel</value>
     <comment>Text for Hours</comment>
   </data>
   <data name="InValidEndTimeMessageAlert" xml:space="preserve">
-    <value>La date de fin que vous avez entrée se produit avant la date de début.</value>
+    <value>Alerte de message de fin non valide</value>
     <comment>Message alert for invalid end time</comment>
   </data>
   <data name="InvalidOccurrenceEditingMessageAlert" xml:space="preserve">
-    <value>Deux occurrences de [] ne peuvent pas se produire le même jour.</value>
+    <value>Alerte de message d'édition d'occurrence non valide</value>
```

Comment:
Check this

Reply:
Changed text to **Alerte de message : édition d'occurrence non valide**